python/cuvs/cuvs/_version.py

```diff
@@ -13,10 +13,22 @@
 # limitations under the License.
 #
 
-
 import importlib.resources
 
 __version__ = (
-    importlib.resources.files("cuvs").joinpath("VERSION").read_text().strip()
+    importlib.resources.files(__package__)
+    .joinpath("VERSION")
+    .read_text()
+    .strip()
 )
-__git_commit__ = ""
+try:
+    __git_commit__ = (
+        importlib.resources.files(__package__)
+        .joinpath("GIT_COMMIT")
+        .read_text()
+        .strip()
+    )
+except FileNotFoundError:
+    __git_commit__ = ""
+
+__all__ = ["__version__", "__git_commit__"]
```

Comment:
```suggestion
__all__ = ["__git_commit__", "__version__"]
```

If you end up pushing another commit here, could you please alphabetize this list? Doesn't matter *that* much when it's only 2 items, but that'd make it easier to view at a glance if it grows to more elements in the future.